mithril-signer/src/entities/signer_epoch_settings.rs

```diff
@@ -8,11 +8,8 @@ pub struct SignerEpochSettings {
     /// Current Epoch
     pub epoch: Epoch,
 
-    /// Current Protocol parameters
-    pub protocol_parameters: ProtocolParameters,
-
-    /// Next Protocol parameters
-    pub next_protocol_parameters: ProtocolParameters,
+    /// Signer registration protocol parameters
+    pub signer_registration_protocol_parameters: ProtocolParameters,
```

Comment:
Since we are in the mithril-signer I think that we can omit the `signer_` prefix (same for the epoch service).
```suggestion
    /// Registration protocol parameters
    pub registration_protocol_parameters: ProtocolParameters,
```

Reply:
You're totally right, I'll make the change.

Reply:
I'm just wondering, since there are also signature registration, is it worth keeping `signer_` as prefix?

Reply:
I made the modification in the following commit: https://github.com/input-output-hk/mithril/pull/2058/commits/f775ec4ad80cf8fc9e31f7238a6d0a705eefbb7d

Reply:
Maybe we want to keep the `signer_` prefix to avoid confusion with the signature registration